modules/effects/src/main/scala/dev/profunktor/redis4cats/effects.scala

```diff
@@ -109,7 +108,7 @@ object effects {
     def idleTime(idleTime: Long): RestoreArgs  = copy(idleTime = Some(idleTime))
   }
 
-  case class ScanArgs(`match`: Option[String], count: Option[Long]) {
+  sealed abstract class ScanArgs(`match`: Option[String], count: Option[Long]) {
```

Comment:
Any reason for this change?
This would break binary compatibility AFAIK

Reply:
Sorry , this was carry over from somewher else , will delete

Reply:
done